ui/page/playlists/internal/collectionsListMine/view.jsx

```diff
@@ -29,7 +29,7 @@ type Props = {
   areBuiltinCollectionsEmpty: boolean,
```

Comment:
The sorting order seems broken.  The one in `master` makes more sense.  This is from the same account.

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/64950861/189804825-b5c3f957-d72d-4fb1-9790-1de2774c0f6b.png">